.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "test-child-repo"]
```

Comment:
I think it would be nice to set up a custom testing repo for this instead of adding submodules. But I will take care of this part, since I also think they should be in the same organization :)

Reply:
Removed submodules and added more effective test. Also fixed some issues that popped up when this macro is ran in a project with no submodules.